README.md

```diff
@@ -238,6 +240,11 @@ test and produce json result files.
 There are some utilities for working with the json result files, run the `performance` script
 with no options to see them.
 
+## Availability Monitor 
+Monitor class aims at verifying availability of overall accumulo cluster by continually doing
+scans of random values across various tablet servers and capturing timing
+information related to how long such scans take. 
+
```

Comment:
Could add an example command showing how to run.

Reply:
This will follow exact same pattern as cingest rwalk and other scripts. I have made the edits in the readme that show this like on Line 47 & Line 35